designs/tripleo/osc-tripleo.md

```diff
@@ -119,7 +119,16 @@ outputs:
               - OSC_ENV_VAR={get_param: OSCEnvVar}
               - TripleO::Services::OSC::tls_truststore_file: {get_param:OSCTrustStoreFile}
               - TripleO::Services::OSC::keystore_file: {get_param:OSCKeyStoreFile}
-```  
+        tripleo.osc.firewall_rules:
```

Comment:
I guess these are the ports on the hosting machine. The ports in the container for OSC shall be hard fixed?